ras/servicelog.py

```diff
@@ -118,14 +118,14 @@ def test_servicelog_notify(self):
         self.log.info("=====6 - Checking if servicelog_notify --list, "
                       "lists the command just added  =======")
         for id in ids:
-            if id:
+            if type(id) == int:
```

Comment:
weird  why this check needed

Reply:
Because it is taking one empty line while am fetching id.so i kept a condition like if that is int then only it should execute the statements

Reply:
yes, I understand that but checking this way not looks good, please strip that before processing it 

it can be done here 
`https://github.com/avocado-framework-tests/avocado-misc-tests/blob/dbd2dc7a8bf54a70027d8d99f84d9076439ca4f0/ras/servicelog.py#L115`